docs: Scrub documentation for Release 0.4.0 (#1053)
bpfman/bpfman: docs/getting-started/example-bpf-k8s.md

> This section will describe loading bytecode on a Kubernetes cluster and launching the userspace
> program.

Reviewer: ```suggestion
This section will describe launching eBPF enabled applications on a Kubernetes cluster.
```

Author: Done

make run-on-kind
```

> ## Loading eBPF Bytecode On Kubernetes

Reviewer: ```suggestion
## Loading eBPF Programs On Kubernetes
```

Author: Done

Instead of using the userspace program or CLI to load the eBPF bytecode as done in previous sections,
the bytecode will be loaded by creating a Kubernetes CRD object.
There is a CRD object for each eBPF program type bpfman supports.

* FentryProgram CRD: [Fentry Sample yaml](https://github.com/bpfman/bpfman/blob/main/bpfman-operator/config/samples/bpfman.io_v1alpha1_fentry_fentryprogram.yaml)
* FexitProgram CRD: [Fexit Sample yaml](https://github.com/bpfman/bpfman/blob/main/bpfman-operator/config/samples/bpfman.io_v1alpha1_fexit_fexitprogram.yaml)

> * KprobeProgram CRD: [Kprobe Examples yaml](https://github.com/bpfman/bpfman/tree/main/examples/config/base/go-kprobe-counter/bytecode.yaml)

Author: @astoycos I don't care either way, but all the remaining links were from the `Examples` directory, which is why I used the word "Examples". And once I write an Fentry/Fexit example, I'll move those as well.

> * KprobeProgram CRD: [Kprobe Examples yaml](https://github.com/bpfman/bpfman/tree/main/examples/config/base/go-kprobe-counter/bytecode.yaml)

Reviewer: Fair enough

    type: Loaded
```

> ## Loading Userspace Container On Kubernetes

Author: Done
